warehouse/templates/includes/input-recaptcha.html

```diff
@@ -0,0 +1,29 @@
+{% macro recaptcha_html() -%}
+  {% if request.find_service(name="recaptcha").enabled %}
+    <div id="recaptcha-container" class="g-recaptcha"></div>
+    {% if form.g_recaptcha_response.errors %}
+    <ul class="errors">
+      {% for error in form.g_recaptcha_response.errors %}
+      <li>{{ error }}</li>
+      {% endfor %}
+    </ul>
+    {% endif %}
+  {% endif %}
+{%- endmacro %}
+
+{% macro recaptcha_src() -%}
+  {% if request.find_service(name="recaptcha").enabled %}
+    <script type="text/javascript">
+      var config = new Object();
+      config.sitekey = "{{ request.registry.settings["recaptcha"]["site_key"] }}";
```

Comment:
Whoops, something I totally intended to change before pushing for review, thanks for the catch.

Why use anonymous functions? FWIW out of habit, I never use anonymous functions anymore because of browser-based debugging. Chrome developer tools would trace [anonymous] or something similar rather than something meaningful. So even when defining a method bound to an object, I'd do something like:

    myobject.prototype = {
        mymethod: function MYOBJECT_mymethod() {
            // do something magical
        }
    };

Maybe things have gotten a little better since I last did anything full stack, but that's what I remember doing in the past.